godot-ffi/src/godot_ffi.rs

```diff
@@ -554,6 +602,9 @@ mod scalars {
     impl_godot_marshalling!(i8 as i64);
     impl_godot_marshalling!(u8 as i64);
 
+    // Some places Godot will pass along 64-bit numbers that are intended to be interpreted as unsigned
+    // integers despite Godot integers being signed by default.
+    impl_godot_marshalling!(u64 as i64; lossy);
```

Comment:
Regarding this, I will open an issue, so we don't forget to remove it from being passed via `#[func]`.